app.rb

```diff
@@ -0,0 +1,80 @@
+# HARDER User Stories:
+# If signed in, don't give "Sign Up" or "Sign In" options
+# Make sure paths flow and edit CSS
+
+require 'sinatra/base'
+require 'sinatra/flash'
+require_relative './lib/user.rb'
+require_relative './database_connection.rb'
+
+class Chitter < Sinatra::Base
```

Comment:
Which part of the controller felt too bloated to you? It feels ok to me, but perhaps you have an instinct or idea that something should be pulled out into a model?

Reply:
It was the post '/save_user' (row 42) and post '/sessions' (row 62) that just felt a bit long, but I'm not sure what could be pulled out into a model so perhaps they're okay